packages/picasso.js/src/core/component/component-factory.js

```diff
@@ -431,8 +447,23 @@ function componentFactory(definition, context = {}) {
   let currentNodes;
   let preComputedRect;
 
+  function updateBrushNodes(nodes) {
```

Comment:
This is needed to support brushing otherwise brushing only hold the nodes are rendered the last time during progressive rendering.